syntax/abstract.mjs

```diff
@@ -82,6 +82,14 @@ export function list_into(Type) {
 
 export function into(Type) {
     switch (Type) {
+        case FTL.Comment:
+        case FTL.GroupComment:
+        case FTL.ResourceComment:
+            return content =>
+                // Only trim if the comment ends with an EOL, not EOF.
+                content.endsWith("\n")
```

Comment:
We should do all three line-ending-variants here?

Reply:
Ah, good point, thanks!